k8s/prod/common/idam/idam-api.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: idam-api
+  namespace: idam
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: idam-api
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: idam-api
+    version: 0.2.4
+  values:
+    java:
+      image: hmctspublic.azurecr.io/idam/api:latest
+      replicas: 2
+      ingressHost: idam-api-prod2.platform.hmcts.net
+      aadIdentityName: idam
+      environment:
+        TESTING_SUPPORT_ENABLED: true
```

Comment:
do you want this on in prod?

Reply:
Whilst prod2 is deployed and not active, we plan to test the service and then remove the test data. Once this is complete, testing support will be disabled.